provider/README.md

```diff
@@ -178,12 +178,40 @@ In addition to the standard [Provider payload wrapper](#response-format), respon
 }
 ```
 
-**Endpoint:** `/vehicles`  
+The `/vehicles` endpoint returns the specified vehicle (if a device_id is provided) or a list of known vehicles.
+
+**Endpoint:** `/vehicles/{device_id}`  
```

Comment:
@marie-x Added individual vehicle lookup option to Provider, like in Agency. Does this work and make sense?

Reply:
yep!